controllers/commentsController/getAllComments.js

```diff
@@ -17,39 +17,16 @@ const responseHandler = require("../../utils/responseHandler");
  */
 const getAllComments = async (req, res, next) => {
   const { applicationId } = req.token; //this will be retrieved from decoded api token after full auth implementation
-  const { refId, origin, ownerId, isFlagged, limit, sort, page } = req.query;
+  const { refId, origin, ownerId, isFlagged } = req.query;
```

Comment:
It might be cleaner to destructure paginateOptions here as well

Reply:
But what's the point if I won't be reusing the destructured variables in more than one place?